Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM php:7.2-fpm-alpine
+
+# Use the default production configuration
+RUN mv "$PHP_INI_DIR/php.ini-production" "$PHP_INI_DIR/php.ini"
+
+RUN apk add --no-cache curl git tar unzip libpng-dev libxml2-dev
+
+RUN docker-php-ext-install bcmath && \
```

Comment:
You can actually replace a lot of this with one line:

`docker-php-ext-install bcmath ctype json gd mbstring pdo pdo_mysql tokenizer xml`